packages/cashier-v2/src/lib/Transfer/components/TransferReceipt/TransferReceipt.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Button, Loader, Text, useDevice } from '@deriv-com/ui';
+import { ReceiptScreen } from '../../../../components';
+import { TTransferReceipt } from '../../types';
+import { getTransferAccountName } from '../../utils';
+import { TransferAccountIcon } from '../TransferAccountIcon';
+import styles from './TransferReceipt.module.scss';
+
+const TransferReceipt: React.FC<TTransferReceipt> = ({ amount, fromAccount, toAccount }) => {
+    const { isMobile } = useDevice();
+
+    if (!amount || !fromAccount || !toAccount) return <Loader />;
+
+    return (
+        <ReceiptScreen
+            actionButtons={
+                <div className={styles.actions}>
+                    <Button size='lg' textSize='sm' variant='outlined'>
+                        View transaction details
+                    </Button>
+                    <Button size='lg' textSize='sm'>
+                        Make a new transfer
+                    </Button>
+                </div>
+            }
+            amount={`${amount} ${fromAccount?.currency}`}
+            fromElement={
+                <>
+                    <div className={styles['account-info']}>
+                        <TransferAccountIcon account={fromAccount} />
+                        <Text weight='bold'>{getTransferAccountName(fromAccount)}</Text>
+                    </div>
+                    <div className={styles['account-id']}>
+                        <Text color='less-prominent' size={isMobile ? 'md' : 'sm'}>
```

Comment:
I think size will cause a glitch in tablet view.

Reply:
Hi @thisyahlen-deriv,
As of now, I don't think we have a design for tablet view cashier legacy transfer.
Will discus this with the designers. Thank you for pointing out.